sensing/ahrs_ekf.hpp

```diff
@@ -134,6 +134,9 @@ class Ahrs_ekf : public Kalman<7, 0, 3>
     Mat<3,3> R_acc_;                                    ///< The accelerometer measruement noise matrix
     Mat<3,3> R_mag_;                                    ///< The magnetometer measurement noise matrix
 
+    float dt_s_;
```

Comment:
missing doc + again for precision, I am not sure it is a good point to go for sec instead of us
to be discussed